<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TerminalWidget</class>
 <widget class="QWidget" name="TerminalWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>942</width>
    <height>512</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="copyButton">
       <property name="text">
        <string>Copy</string>
       </property>
       <property name="icon">
        <iconset theme="edit-copy"/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pasteButton">
       <property name="text">
        <string>Paste</string>
       </property>
       <property name="icon">
        <iconset theme="edit-paste"/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="intButton">
       <property name="text">
        <string>SIGINT</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="tstpButton">
       <property name="text">
        <string>SIGTSTP</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="quitButton">
       <property name="text">
        <string>SIGQUIT</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
